lib/rack/prx_auth/version.rb

```diff
@@ -1,5 +1,7 @@
+require 'prx_auth/version'
```

Comment:
rack-prx_auth still needs its own version?

Reply:
Not really, just wasn't sure if something else was expecting it, but we can change it if so. Removed.